examples/framework/preferences/shared/html/incSize-template.html

```diff
@@ -1,10 +1,16 @@
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="incSize-choice" class="mpe-incSize" />
     <label for="incSize-choice">
-    	<!-- FLUID-5708 Description text must be before toggle to prevent overflow issue -->
-    	<span class="mpe-incSize-descr"></span>
-        <span class="fl-prefsEditor-switch" data-checkboxStateOn="ON" data-checkboxStateOff="OFF">
-            <span class="fl-prefsEditor-switch-inner"></span>
+        <span class="mpe-incSize-descr fl-prefsEditor-panel-descr"></span>
+        <span class="fl-prefsEditor-switch fl-input-enhance" aria-hidden=true>
+            <span class="fl-prefsEditor-switchOn">
+                <span class="fl-prefsEditor-switch-indicator">✔</span>
+                <span class="flc-prefsEditor-switchOn-label fl-prefsEditor-switchOn-label">ON</span>
+            </span>
+            <span class="fl-prefsEditor-switchOff">
+                <span class="fl-prefsEditor-switch-indicator">✔</span>
+                <span class="flc-prefsEditor-switchOff-label fl-prefsEditor-switchOff-label">OFF</span>
```

Comment:
Is "fl-prefsEditor-switchOff-label" and "fl-prefsEditor-switchOn-label" necessary? They could be the same class since they take the same styles. 

The parent fl-prefsEditor-switchOff and fl-prefsEditor-switchOn selector could be used to select specific on or off labels anyway.

This comment is duplicated in other relevant files.